sendgrid_sender.go

```diff
@@ -73,6 +73,10 @@ func buildMail(m mail.Message) (*smail.SGMailV3, error) {
 		p.AddTos(smail.NewEmail(to.Name, to.Address))
 	}
 
+	for k, v := range m.Data {
+		p.SetCustomArg(k, fmt.Sprintf("%v", v))
```

Comment:
Here it should be something like:
```go
if customArgs, ok := m.Data[CustomArgsKey].(CustomArgs); ok {
   [iterate through CustomArgs]
}
```
`CustomArgsKey` should be a constant in this package.
`CustomArgs` should be a type `map[string]string`

Reply:
Do you refer to get in m.Data a default key pe. `customArgsKey` to add the custom args?